octoprint_filamentsensorsimplified/static/css/filamentsensorsimplified.css

```diff
@@ -16,5 +16,5 @@
 
 #filamentsensorsimplified_settings_testResult{
 	display: block;
-	margin-bottom: 10px;
+	float: right;
```

Comment:
Why do you think having this floating on the right is better idea?
![image](https://user-images.githubusercontent.com/13912644/131744178-c6376ec4-4720-4664-9e9a-886612b39b5f.png)

Reply:
I don't i will change it back